index.js

```diff
@@ -70,6 +70,7 @@ class WebTorrent extends EventEmitter {
     this.torrentPort = opts.torrentPort || 0
     this.dhtPort = opts.dhtPort || 0
     this.tracker = opts.tracker !== undefined ? opts.tracker : {}
+    this.lsd = opts.lsd === true
```

Comment:
If no `opts.lsd` value is specified, then this will be `false`. I think you meant:

```js
opts.lsd !== false
```

Reply:
another nifty trick is double negation: `!!opts.lsd`

Reply:
@justchen1369 Yours is a bit different – it coerces the argument to a boolean, then inverts it. In this case, doesn't matter much. But in general, I lean towards explicitness. If they pass anything other than `false`, it should just stick with the default, which is `true`.

Reply:
Fixed the way @feross suggested 👍